### PR TITLE
Adjust text on Antigen pending and negative test result screens

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Adjust test on negative test result and pending test result screens in antigen tests

## Link to Jira
pending screen ticket: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7578
negative screen ticket: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7608

## Screenshots
IMG_0087
IMG_0088

### CODE REVIEW

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift [hunk 1]
```diff
@@ -221,36 +221,64 @@ extension ExposureSubmissionTestResultViewModel {
...
+					cells.append(
+						ExposureSubmissionDynamicCell.stepCell(
+							title: AppStrings.ExposureSubmissionResult.testCertificateTitle,
+							description: AppStrings.ExposureSubmissionResult.Antigen.testCertificateNotRequested,
+							icon: UIImage(named: "certificate-qr-light"),
+							hairline: .none
+						)
+					)
+				}
+			}
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift:281]:
```suggestion
				} else if !test.certificateConsentGiven {
					cells.append(
						ExposureSubmissionDynamicCell.stepCell(
							title: AppStrings.ExposureSubmissionResult.testCertificateTitle,
							description: AppStrings.ExposureSubmissionResult.Antigen.testCertificateNotRequested,
							icon: UIImage(named: "certificate-qr-light"),
							hairline: .none
						)
					)
				} else if !test.certificateRequested {
					cells.append(
						ExposureSubmissionDynamicCell.stepCell(
							title: AppStrings.ExposureSubmissionResult.testCertificateTitle,
							description: AppStrings.ExposureSubmissionResult.Antigen.testCertificatePending,
							icon: UIImage(named: "certificate-qr-light"),
							hairline: .none
						)
					)
				}
			}
```

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift:281]: I cant dind the property `test.certificateRequested`  in it inside the **AntigenTest** struct ?

Screenshot 2021-06-07 at 09 44 38

Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift:281]: Sorry, on your branch it's called `certificateCreated`, I just refactored it on mine to make the naming more accurate. You can just use `certificateCreated` here

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by nickguendling — APPROVED** [review]

**Pull request merged**